### DISCUSSION TITLE
Unity App for iOS build fails

### DISCUSSION THREAD

**Discussion by AbrarZShahriar** [keygen-sh/community#75] [Issue]
Hello all,
I used Nuget to install 
this BouncyCaslte on Unity https://www.nuget.org/packages/Portable.BouncyCastle
For Nsec I used this https://github.com/Kirbyrawr/nsec-unity as the one from Nuget was failing at runtime. 

The Unity project builds and runs fine on Windows and Android but fails on iOS. 

Undefined symbols.

How can I fix it? Thanks

**Comment by AbrarZShahriar** [comment]
Apparently Nsec.cryptography lib uses libsodium and iOS is not supported... 

Anyway I could easily get around the issue?

**Comment by ezekg** [comment]
This doesn't seem to be an issue with Keygen itself. So I'd open up an issue on these repos and see if your errors can be resolved. Alternatively, you can switch to a crypto library with iOS support, or perhaps implement [Apple CryptoKit](https://developer.apple.com/documentation/cryptokit/) on iOS specifically.
